tool/src/test/java/org/wildfly/security/tool/FileSystemRealmCommandTest.java

```diff
@@ -247,25 +221,62 @@ private void compareScriptOutput(String fileSystemRealmName, String securityDoma
         assertEquals(scriptLines.get(1), expectedSecurityDomainScript.toString());
     }
 
+    @BeforeClass
+    public static void getAbsolutePaths() throws Exception {
+        String OS = System.getProperty("os.name").toLowerCase();
+        absoluteBaseDir = Paths.get(FileSystemRealmCommandTest.class.getProtectionDomain().getCodeSource().getLocation().toURI()).toAbsolutePath().toString();
+        if (OS.contains("win")) {
+            absoluteBaseDir += "\\filesystem-realm\\";
```

Comment:
Maybe use File.separator here

Reply:
Changed it to use System.getProperty("file.separator")